misk/src/main/kotlin/misk/web/actions/WebActionEntry.kt

```diff
@@ -18,8 +17,14 @@ data class WebActionEntry(
   val pathPrefix: String = ""
 ) {
   init {
-    require(pathPrefix.matches(Regex("(/[^/]+)*"))) {
+    require(pathPrefix.matches(Regex("(/[^/]+)*")) && !pathPrefix.startsWith("/api")) {
```

Comment:
Why the special call-out for `/api` ? What implication does this have for the URI space available for application installed actions?

Reply:
Good catch, only meant that to be a requirement on WebProxyEntry. Purpose was to encourage an idiom that web facing endpoints use prefix "/api". This then prevents having a URI prefix that matches on a WebProxyEntry which would forward the entry to a build server instead of Misk handling it.